stats/stats_test.go

```diff
@@ -29,6 +29,11 @@ import (
 	"fortio.org/fortio/log"
 )
 
+const (
+	ipOne = "127.0.0.1"
+	ipTwo = "127.0.0.2"
```

Comment:
We had talked about but I forgot to set it up until I saw this - I just added:
```
% host ltest.fortio.org
ltest.fortio.org has address 127.0.0.2
ltest.fortio.org has address 127.0.0.3
ltest.fortio.org has address 127.0.0.1
```

Reply:
note for anyone looking at this later

on linux a fortio server does bind on all the 127.0.0.x addresses but not on macos